File: python_programming_basics/conditional_statements _if_else_elif_debug/l_07_area_of_figures.py
from math import pi

figure = input()
area = ""

if figure == "square":
    a = float(input())
    area = a * a

elif figure == "rectangle":
    a = float(input())
    b = float(input())

    area = a * b

elif figure == "circle":
    radios = float(input())

    area = pi * radios ** 2

elif figure == "triangle":
    a = float(input())
    b = float(input())

    area = a * b / 2

print(f'{area:.3f}')

File: JavaScript Basics/02. Conditional Statements/l-07-Area-of-Figures.js
function solve(arg1, arg2, arg3) {
    let type = arg1;
    
    if (type === "square") {
        let a = Number(arg2);
        let result = a * a;
        console.log(result.toFixed(3));

    } else if (type === "rectangle") {
        let length = Number(arg2);
        let width = Number(arg3);
        let result = length * width;
        console.log(result.toFixed(3));

    } else if (type === "circle") {
        let radius = Number(arg2);
        let result = Math.PI * radius * radius;
        console.log(result.toFixed(3));

    } else if (type === "triangle") {
        let base = Number(arg2);
        let height = Number(arg3);
        let result = (base * height) / 2;
        console.log(result.toFixed(3));
    }
}

solve()
